### ISSUE TITLE
Temporal lobe calcification

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7568]
**Preferred term label:**
Temporal lobe calcification

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of calcium deposition within the temporal lobe of the brain.
PMID:20027545

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 247100 
PMID: 12603844  3/10

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
